docs/setting-up/server/dbaas.md

```diff
@@ -42,7 +42,7 @@ alias kubectl='minikube kubectl --'
 ## Start PMM server and activate a DBaaS feature
 
 !!! note alert alert-primary ""
-    - To start a fully-working 3 node XtraDB cluster, consisting of sets of 3x HAProxy, 3x PXC and 6x PMM Client containers, you will need at least 9 vCPU available for minikube. (1x vCPU for HAProxy and PXC and 0.5vCPU for each pmm-client containers).
+    - To start a fully-working 3 node XtraDB cluster, consisting of sets of 3x HAProxy, 3x PXC and 6x PMM Client containers, you will need at least 9 vCPU available for minikube. (0.5vCPU for HAProxy and PXC and 0.5vCPU for each pmm-client containers).
```

Comment:
To start a fully-working 3 node PXC cluster, consisting of sets of 3x HAProxy, 3x PXC and 6x PMM Client containers, you will need at least 0.5vCPU for HAProxy and 0.5vCPU for each pmm-client containers. CPU for PXC containers is set in UI by user.

Reply:
+1